crates/router/src/core/user.rs

```diff
@@ -644,6 +644,8 @@ async fn handle_existing_user_invitation(
         created_at: now,
         last_modified: now,
         entity: domain::NoLevel,
+        // TODO: To be sent from token after option is removed
+        tenant_id: state.tenant.tenant_id.clone(),
```

Comment:
Can we do `unwrap_or` instead.

This way compiler tells us to change once we remove `tenant_id` from the JWT.

Reply:
Yes, that would be better. Thanks